docs/content/tutorial/default.yml

```diff
@@ -207,7 +207,7 @@ body:
         "name": "Nico Williams"
       }
       {
-        "message": "Fix iterration problem for non decimal string\n\nWhen the string transformation to number failed, all following\ntransformation failed too.\n\nThis happend because status in decNumberFromString function is\nupdated just in error case. Reusing the DEC_CONTEXT that failed\nbefore results into error even if the string is valid number.",
+        "message": "Fix iteration problem for non decimal string\n\nWhen the string transformation to number failed, all following\ntransformation failed too.\n\nThis happend because status in decNumberFromString function is\nupdated just in error case. Reusing the DEC_CONTEXT that failed\nbefore results into error even if the string is valid number.",
```

Comment:
Are you aware that this is a commit message of this repository? Fixing typos like this example looks not just meaningless, but lacks well investigation of the context.

Reply:
I'm not trying to waste anybody's time, I saw a typo in an OSS project that I like and I fixed it.
I agree that this PR is far less "meaningful" now that the decNumber fixes are out of scope - feel free to close it.